doc/api/toolkits/mplot3d.rst

```diff
@@ -1,3 +1,34 @@
+.. _toolkit_mplot3d-index:
+.. currentmodule:: mpl_toolkits.mplot3d
+
+*******
+mplot3d
+*******
+
+The mplot3d toolkit adds simple 3D plotting capabilities (scatter, surface,
+line, mesh, etc.) to Matplotlib by supplying an axes object that can create
+a 2D projection of a 3D scene.  The resulting graph will have the same look
+and feel as regular 2D plots.  Not the fastest or most feature complete 3D
+library out there, but it ships with Matplotlib and thus may be a lighter
+weight solution for some use cases.
+
+See the :doc:`mplot3d tutorial </tutorials/toolkits/mplot3d>` for
+more information.
+
+.. image:: /_static/demo_mplot3d.png
+   :align: center
+
+The interactive backends also provide the ability to rotate and zoom
+the 3D scene. One can rotate the 3D scene by simply clicking-and-dragging
+the scene. Zooming is done by right-clicking the scene and dragging the
+mouse up and down. Note that one does not use the zoom button like one
+would use for regular 2D plots.
```

Comment:
This is unclear - is it tied to the sentence before it?

Reply:
reworded